src/Components/CategoryMovies/CategoryMovies.js

```diff
@@ -0,0 +1,89 @@
+import "./categoryMovies.css";
+import Movie from "./../Movie/Movie";
+import { useProducts } from "./../../Context/products-context.js";
+import { useVideos } from "./../../Context/videos-context";
+import { useEffect } from "react";
+
+const CategoryMovies = ({ category }) => {
+  const { stateVideo, dispatchVideo } = useVideos();
+
+  useEffect(() => {
+    fetch("/api/videos", {
```

Comment:
I would recommend you to use axios it makes are life a lot easier.

Reply:
yeah, try Axios for API calls.